Content.IntegrationTests/Tests/Storage/StorageInteractionTest.cs

```diff
@@ -0,0 +1,75 @@
+using Content.Client.UserInterface.Systems.Hotbar.Widgets;
+using Content.Client.UserInterface.Systems.Storage.Controls;
+using Content.IntegrationTests.Tests.Interaction;
+using Content.Shared.Input;
+using Content.Shared.PDA;
+using Content.Shared.Storage;
+using Robust.Client.UserInterface;
+using Robust.Shared.Containers;
+using Robust.Shared.GameObjects;
+
+namespace Content.IntegrationTests.Tests.Storage;
+
+public sealed class StorageInteractionTest : InteractionTest
+{
+    /// <summary>
+    /// Check that players can interact with items in storage if the storage UI is open
+    /// </summary>
+    [Test]
+    public async Task UiInteractTest()
+    {
+        var sys = Server.System<SharedContainerSystem>();
+
+        await SpawnTarget("ClothingBackpack");
+        var backpack = ToServer(Target);
+
+        // Initially no BUI is open.
+        Assert.That(IsUiOpen(StorageComponent.StorageUiKey.Key), Is.False);
+        Assert.That(IsUiOpen(PdaUiKey.Key), Is.False);
+
+        // Activating the backpack opens the UI
+        await Activate();
+        Assert.That(IsUiOpen(StorageComponent.StorageUiKey.Key), Is.True);
+        Assert.That(IsUiOpen(PdaUiKey.Key), Is.False);
+
+        // Pick up a PDA
+        var pda = await PlaceInHands("PassengerPDA");
```

Comment:
these should be either static proto ids here or dummy prototypes defined herer

Reply:
If you mean make them static EntProtoId fields so they get linted, that doesn't really matter as the test already implicitly checks that they're valid. If you just mean make them static, but I'm also not sure if that'd be useful if they only get referenced a single time.

I'm reluctant to use dummy protos because part of the functionality of the test is to ensure that nothing has unexpectedly broken backpacks. E.g., using a dummy proto wouldn't catch issues due to someone adding some bad component to the `Clothing` parent prototype that somehow causes interactions to stop working.

Reply:
its less for linting and more that it helps bulk-renaming cuz then you can just grep for `EntProtoId.*PassengerPDA` but yeah, it isnt that necessary